wls-common/security/src/main/java/de/muenchen/oss/wahllokalsystem/wls/common/security/domain/BezirkUndWahlID.java

```diff
@@ -0,0 +1,41 @@
+package de.muenchen.oss.wahllokalsystem.wls.common.security.domain;
+
+import java.io.Serializable;
+import java.util.Objects;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
```

Comment:
`@Getter` und `@Setter` ist Teil von [`@Data`](https://projectlombok.org/features/Data)

```suggestion
import java.io.Serializable;
import java.util.Objects;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
```